File: cypress/support/scripts/pyfaas4i/functions/download-project.py
from pyfaas4i import faas
import sys, json

path = sys.argv[1]
filename = sys.argv[2]

is_local_var = False

while is_local_var == False:
    projects = faas.list_projects(version_check=False)
    for row in projects.itertuples(index=True, name='Pandas'):
        if(row.status == "success"):
            project_success = row.id
            is_local_var = "project_success" in locals()


print(faas.download_zip(project_success,path,filename, version_check=False))

File: cypress/support/scripts/faas4i/functions/download-project.R
library(faas4i)

myargs = commandArgs(trailingOnly=TRUE)

while(!exists("project_success")){
    projects <- faas4i::list_projects(version_check = F)
    for(i in 1:length(projects)){
        if(projects[[i]][["status"]] == 'success'){
            project_success <-projects[[i]][["id"]]
        }
    }
}

faas4i::download_zip(project_id = project_success[1], path = myargs[1], filename = myargs[2], version_check = F)